### PR TITLE
fix(swap): Fix HW wallet fee info

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves [YOMO-863](https://emurgo.atlassian.net/jira/software/c/projects/YOMO/issues/YOMO-863)
Screenshot 2023-10-12 at 11 49 32

[YOMO-863]: https://emurgo.atlassian.net/browse/YOMO-863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx [hunk 1]
```diff
@@ -29,7 +37,7 @@ export const TransferSummary = ({wallet, unsignedTx}: {wallet: YoroiWallet; unsi
         <Text>{strings.fees}</Text>
 
         <Text style={styles.balanceAmount} testID="feeAmountText">
-          {formatTokenWithText(feeAmount.quantity, wallet.primaryToken)}
+          {additionalFees ? `${additionalFees} + ${transactionFee}` : transactionFee}
```
Comment (michalszorad) [inline apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx]: The fee is displayed as `0.95 ADA + tx fee in ADA`

File: apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx [hunk 2]
```diff
@@ -29,7 +37,7 @@ export const TransferSummary = ({wallet, unsignedTx}: {wallet: YoroiWallet; unsi
...
         </Text>
       </Item>
```
Comment (michalszorad) [inline apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx:44]: The final balance however does not seem correct, or is it? See screenshot in description

Comment (stackchain) [reply apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx:44]: @MichalSzorad do we need this fee there, it looks legacy from the withdraw, cuz there is no recovered balance, and we have the TxSummary that already displayed everything including the txFee, can you check with product? in the new Design (MAT confirmation funnel) only the CTA is there.

Comment (michalszorad) [reply apps/wallet-mobile/src/Dashboard/WithdrawStakingRewards/TransferSummary/TransferSummary.tsx:44]: I'll change this to match MAT

**Review by banklesss — APPROVED** [review]

**Comment by michalszorad** [comment]
Closing in favour of https://github.com/Emurgo/yoroi/pull/2764

**Pull request closed without merging**